backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -206,11 +205,16 @@ CREATE TABLE accounts(
         -- Starting at 1 to count the transaction that made the account.
         DEFAULT 1,
     -- The total delegated stake of this account in micro CCD.
-    -- TODO: Actually populate this in the indexer.
     delegated_stake
         BIGINT
         NOT NULL
-        DEFAULT 0
+        DEFAULT 0,
+    delegated_restake_earnings
+        BOOLEAN
+        NULL,
+    delegated_target_baker_id
+        BIGINT
+        NULL
```

Comment:
Please add documentation for these columns, it should include when the values would be NULL